sup3r/pipeline/forward_pass.py

```diff
@@ -440,6 +449,16 @@ def get_hr_slices(slices, enhancement, padding, step=None):
             hr_slices.append(slice(start, stop, step))
         return hr_slices
 
+    @property
+    def n_spatial_chunks(self):
+        """Get the number of spatial chunks"""
+        return len(self.hr_crop_slices[0])
```

Comment:
So is the default now to distribute all the spatial chunks to individual nodes? Is there any control over this besides the fwp chunk shape?